odtp/cli/new.py

```diff
@@ -77,8 +95,28 @@ def digital_twin_entry(
 
     if user_email:
         user_id = db.get_document_id_by_field_value("user_email", user_email, "users")
-
+    
     dt_id = db.add_digital_twin(userRef=user_id, name=name)
+
+    dt_doc = db.get_document_by_id(
```

Comment:
These db calls belong to `generate_digital_twin_path`: add them there: these functions should be black boxes, that know themselves what to do.